README.md

```diff
@@ -60,9 +60,9 @@ Then place the msInfiniteScroll directive on the `mat-select` component
 
 ### Compatibility
 
-* `@angular/core`: `>=6.0.0 <16`,
-* `@angular/cdk`: `>=6.0.0 <16`,
-* `@angular/material`: `>=6.0.0 <16`,
+* `@angular/core`: `>=16 <16`,
```

Comment:
I believe this should be `>=15 <16`

Reply:
@Brachacz corrected, hope the author will take a look , looks like this package is no longer maintained not sure